app/views/favourites/index.html.slim

```diff
@@ -0,0 +1,5 @@
+h1 Your Favourite Products
+
+- @favourites.each do |favourite|
+  div
+    = link_to favourite.post.title, post_path(favourite.post)
```

Comment:
In order to make sure it doesn't throw an exception when a user doesn't have any favourite post
<% current_user.favorite_posts.each do |favpost| %>
  <li><%= link_to favpost.title, post_path(favpost) %></li>
<% end %>